src/Processors/Transforms/FilterTransform.cpp

```diff
@@ -182,6 +188,14 @@ void FilterTransform::doTransform(Chunk & chunk)
     else
         num_filtered_rows = filter_description->countBytesInFilter();
 
+    ProfileEvents::increment(ProfileEvents::WhereFilterPassedRows, num_filtered_rows);
```

Comment:
We also return above when `constant_filter_description.always_true` is true, so this might not be the single place where you have to update this metric.

Reply:
Done